test/constraints/constraint_unit.jl

```diff
@@ -184,7 +184,7 @@ function test_constraint_units_available()
         ]
         SpineInterface.import_data(url_in; relationships=relationships, object_parameter_values=object_parameter_values)
         m = run_spineopt(url_in; log_level=0, optimize=false)
-        var_units_available = m.ext[:spineopt].variables[:units_available]
+        var_units_available = m.ext[:spineopt].variables[:units_available]        
```

Comment:
Just adding white space?

Reply:
Fixed in [latest commit](https://github.com/spine-tools/SpineOpt.jl/commit/21097e41186dd36e83125e591bcfd34989910eeb)